lib/Dialect/OM/Evaluator/Evaluator.cpp

```diff
@@ -120,6 +121,37 @@ circt::om::Evaluator::instantiate(
   return success(std::shared_ptr<evaluator::ObjectValue>(object));
 }
 
+bool hasOMTarget(StringRef path) {
+  return (path.startswith("OMDontTouchedReferenceTarget") ||
+          path.startswith("OMMemberInstanceTarget") ||
+          path.startswith("OMMemberReferenceTarget") ||
+          path.startswith("OMReferenceTarget"));
+}
+
+/// Parse origTarget and prepend the root path.
+std::string addPathPrefix(StringRef origTarget, StringRef rootPath) {
```

Comment:
I was going to suggest using FIRRTL's `tokenizePath` but I see there are some differences.

Once we pin down how this works (if haven't already), may be good to have this have error paths if things aren't as expected / invalid formats?